### ISSUE TITLE
ghcr.io/alan-turing-institute/guacamole-user-sync:v0.6.0 keeps terminating

### ISSUE THREAD

**Issue by mattwestby** [alan-turing-institute/data-safe-haven#2314]
## :white_check_mark: Checklist

- [x] I have searched open and closed issues for duplicates.
- [ ] This is a problem observed when **deploying** a Data Safe Haven.
- [x] I can reproduce this with the [latest version](https://github.com/alan-turing-institute/data-safe-haven/releases).
- [x] I have read through the [documentation](https://alan-turing-institute.github.io/data-safe-haven/).
- [ ] This isn't an open-ended question (open a [discussion](https://github.com/alan-turing-institute/data-safe-haven/discussions) if it is).

## :computer: System information

- **Operating System:** 
- **Data Safe Haven version:** 5.0.1

## :package: Packages

<details>
<summary>List of packages</summary>

```none
Paste list of packages here
```

</details>

## :no_entry_sign: Describe the problem

The guacamole user sync container ghcr.io/alan-turing-institute/guacamole-user-sync:v0.6.0 keeps terminating within the remote desktop container instance. This happens a couple of minutes after deployment, you can restart the container instance and it runs for a couple of mins then terminates. Unfortunately I cant retrieve any logs as they get wiped on termination.

## :deciduous_tree: Log messages

<details>
<summary>Relevant log messages</summary>

```none
Your log details here
```

</details>

**Comment by JimMadge** [comment]
@jemrobinson I recall we had a similar problem in the past. Wasn't that fixed with a new release though? v0.6.0 is the latest version.

I don't recall seeing this problem recently, have you encountered this in your testing @craddm @helendduncan?

**Comment by JimMadge** [comment]
@mattwestby reported downgrading to v0.5.0 as a workaround.

**Comment by craddm** [comment]
I haven't seen any problems since v0.6.0 - may be a different problem, but hard to diagnose without logs, unfortunately

**Comment by JimMadge** [comment]
Luckily, logs for containers will be in the next release 😅.

**Comment by JimMadge** [comment]
@mattwestby the latest release, v5.2.0, collects the logs for container instance to a central log analytics workspace, so they are retained after a container process ends.

Could you deploy v5.2.0 or upgrade in place to see if you can reproduce this?

**Comment by mattwestby** [comment]
Thanks for the heads up Jim, have you got any release notes for v5.2.0?

Thanks

Matt

**Comment by JimMadge** [comment]
@mattwestby yes, you can see them [here](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.2.0).

**Comment by mattwestby** [comment]
Thanks Jim,

How do we submit feature / change requests for the TRE?

Thanks

Matt

**Comment by JimMadge** [comment]
@mattwestby you could open an issue or discussion in this repo.

**Comment by jemrobinson** [comment]
@mattwestby : This may have been the same issue as #2350. Can you check whether v5.2.1 fixes this issue?
